runtime/tests/integration_test.rs

```diff
@@ -25,22 +25,25 @@ use frame_support::{
 use moonbeam_runtime::{AccountId, Balance, Event, InflationInfo, Range, Runtime, GLMR};
 use sp_runtime::Perbill;
 
-pub type SystemModule = frame_system::Pallet<Runtime>;
-pub type StakingModule = parachain_staking::Pallet<Runtime>;
-
-// fn run_to_block(n: u32) {
-// 	while SystemModule::block_number() < n {
-// 		StakingModule::on_finalize(SystemModule::block_number());
-// 		SystemModule::set_block_number(SystemModule::block_number() + 1);
-// 		StakingModule::on_initialize(SystemModule::block_number());
-// 	}
-// }
+type System = frame_system::Pallet<Runtime>;
+type Staking = parachain_staking::Pallet<Runtime>;
+type AuthorInherent = author_inherent::Pallet<Runtime>;
+type ParachainSystem = cumulus_pallet_parachain_system::Pallet<Runtime>;
```

Comment:
I saw Acala doing this too. But can we import these? (not tested, just thinking)

```Rust
use moonbeam_runtime::{System, ParachainStaking, AuthorInherent, ParachainSystem};
```

Reply:
That does work, nice catch!